crates/bevy_render/src/batching/gpu_preprocessing.rs

```diff
@@ -356,26 +356,7 @@ impl FromWorld for GpuPreprocessingSupport {
 
         // filter some Qualcomm devices on Android as they crash when using GPU preprocessing.
         fn is_non_supported_android_device(adapter: &RenderAdapter) -> bool {
-            if cfg!(target_os = "android") {
-                let adapter_name = adapter.get_info().name;
-
-                // Filter out Adreno 730 and earlier GPUs (except 720, as it's newer than 730)
-                // while also taking suffixes into account like Adreno 642L.
```

Comment:
I think both lines of this comment would be useful to preserve. (but second should be moved to `get_adreno_model`)